docs/stale-subvolume.md

```diff
@@ -0,0 +1,32 @@
+# stale-subvolume cleanup
+
+The subvolume command is used to clean the stale subvolumes
+which has no parent-pvc attached to it.
+The command would list out all such subvolumes which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+The subvolume command will require the following sub commands:
+* `ls` : [ls](#ls) lists all the stale subvolumes that can be removed
+* `clean` : [clean](#clean) stale subvolumes after user's confirmation
+
+## ls
+
+```bash
+kubectl rook-ceph stale-subvolume ls
```

Comment:
Is the intention to list stale subvolumes in all filesystems? That seems reasonable, but we might want an optional argument for the name of a specific filesystem. Also, seems like we need to show which filesystem the volume belongs to.

Reply:
+1

Reply:
yes the output shows the filesystem and subvolumegroup to which the subvolume belongs. But again giving users to add option to pass filesystem name is valid but would consider to take in as another PR/enhancement?